setup.md

```diff
@@ -0,0 +1,334 @@
+
+On linux/macOS this is generally done though a bash/zsh shell.
+
+- In linux this can be accessed by searching for terminal in the startup menu.
+  See below for an example in the ubuntu gnome shell.
+
+![Ubuntu shell](./images/main-setup/ubuntu-terminal.png)
+
+- On macOS this is done by searching for terminal in the Launchpad (see the
+  [following for more details][apple terminal support]).
+
+![macos shell](./images/main-setup/macos-terminal.png)
+
+On Windows (unless using WSL in which case please follow the linux instructions),
+this will be done via the Anaconda prompt. The Anaconda prompt can be opened by
+searching for it in the startup menu after having installed Anaconda (see the
+step above).
+
+![Anaconda Prompt](./images/main-setup/anaconda-terminal.png)
+
+There are substantial differences in the way in which the terminal works between
+Windows and linx/macOS. We will give a quick overview of the the basics below,
+including some links for extra resources you can look at to get more familiar
+with using the shell. We recommend that you have a look through these before
+attempting the tutorials.
+
+Note: in the MD tutorial you will eventually be required to access a remote
+HPC machine to run a Molecular Dynamics simulations. This will require use
+of the linux terminal. If you are using Windows you will therefore also need
+to look at the instructions for the linux terminal before attempting this tutorial.
+
+### Basic linux/macOS terminal instructions
+
+This section only provides a very basic overview of using the bash/zsh shell
+in linux/macOS. For more information please see the following tutorials:
+
+- [Ubuntu's terminal tutorial](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview)
+- [MacOS terminal beginners guide](https://www.youtube.com/watch?v=aKRYQsKR46I) (youtube video)
+
+**_On first opening the terminal_**
+
+Generally upon opening the terminal you will be placed in your $HOME directory.
+This is the base directory where all the documents for your login account are
+placed. If you ever need to return to this directory enter the following command:
+
+```
+cd $HOME
+```
+or even just
+
+```
+cd
+```
+
+
+**_Seeing and navigating the contents of folders_**
+
+One of the main uses of the terminal is to allow users to quickly see and navigate
+through the various files and folders on your computer.
+
+You can see what files are in the current folder you are accessing using:
+
+```
+ls -lrt
+```
+
+You can change from one folder to another (in this case named 'newfolder') by doing:
+
+```
+cd newfolder
+```
+
+If you want to know where you are amongst all your folders you can do:
+
+```
+pwd
+```
+
+This will give you the full path to the current folder.
+
+If you want to delete (remove) a file do:
+
+```
+rm file
+```
+
+If you want to rename a file do:
+
+```
+mv file newfilename
+```
+
+If you want to print the contents of a file do:
+
+```
+more file
+```
+
+
```

Comment:
Is this needed/relevant? I'd avoid adding too much information.

Reply:
I'll change the word `print` to `view`, but I think it's a useful one to have around (there's several parts of the tutorials that require cat/tail/etc.. if I remember right, it'd be useful for folks to be able to quickly view a file & have a general appreciation that this can be done - note this is kinda just straight up taken from the linux hints in the Docking tutorial)

Reply:
> have a general appreciation that this can be done
I understand the feeling, with too much content and information the risk is to loose students. But if this is already elsewhere anyway it's ok then.